client/src/modules/list/components/Items/ListItemDescription/ListItemDescription.jsx

```diff
@@ -87,6 +87,12 @@ class ListItemDescription extends PureComponent {
   };
 
   handleShowTextarea = event => {
+    const { nodeName } = event.target;
+
+    if (nodeName === 'A') {
```

Comment:
what does it do? where is node with name `A`?

Reply:
Item's description can contain links. This div which contains the item's description has the onClick handler attached. On click, it turns on description edit mode, but when the user clicks a link inside this div, edit mode shouldn't be turned on.
Event.target.nodeName is a read-only property and returns the name of the current Node as a string. When the user clicks a link inside this clickable div nodeName will return "A". Thanks to that I can detect that anchor tag was clicked and prevent turning edit mode on. But the link is still working normally.

Reply:
maybe create enum like
const NodeTypes = Object.freeze({
     LINK: 'A'
});

and compare to it